test/functional/cmdLineTests/ignoreUnrecognizedVMOptions/ignoreunrecognizedxxcolonoptions.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2019, 2019 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<suite id="IgnoreUnrecognizedXXColonOptions Command-Line Option Tests" timeout="2400">
+
+ <variable name="IGNOREUNRECOGNIZEDXXOPTIONS" value="-XX:+IgnoreUnrecognizedXXColonOptions" />
+ <variable name="NOIGNOREUNRECOGNIZEDXXOPTIONS" value="-XX:-IgnoreUnrecognizedXXColonOptions" />
+
+ <variable name="BADOPTION1" value="-XX:INVALID" />
+ <variable name="BADOPTION2" value="-XX:-INVALID" />
+ <variable name="BADOPTION3" value="-XX:+INVALID" />
+ <variable name="GOODOPTION" value="-XX:+DisableExplicitGC" />
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:INVALID">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION1$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:-INVALID">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION2$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:+INVALID">
```

Comment:
We don't need so many tests. They do take some time to run. Let's remove all the single option variants like this one, and only keep the double option tests like `-XX:-IgnoreUnrecognizedXXColonOptions -XX:+IgnoreUnrecognizedXXColonOptions`